### ISSUE TITLE
Eclairage

### ISSUE THREAD

**Issue by antoine-tesson** [cnigfr/schema-accessibilite-voirie#53]
Je pense qu'il ne serait pas inutile de revenir sur l'attribut éclairage. Voici les attributs possibles du standard : 
01 adapté pour les déficients visuels 
02 éclairé mais non adapté pour les déficients visuels
03 non éclairé

J'avoue ne pas vraiment savoir comment faire une différence entre le 01 et 02 notamment quand le diagnostic s'effectue en journée. Dans la grande majorité des cas, les rues sont éclairées assez convenablement pour la plupart des gens. Pour les malvoyants la perception du niveau d'éclairage est multifactorielle. Plus le revêtement est mauvais, plus le trottoir est étroit, plus il y a d'obstacles et plus le besoin d'éclairage se fait sentir. (C'est d'ailleurs vrai pour tout le monde dans des proportions moins importantes).

Le côté anxiogène de la nuit chez certains peut aussi donner le sentiment qu'une rue n'est pas suffisamment éclairée. L'éclairage public a donc cette double fonction de permettre de voir mais également de rassurer. C'est vrai pour chacun d'entre nous mais cela peut être décuplé chez les enfants, les personnes en situation de handicap mental, les personnes ayant des troubles psychiques et certain.es diront aussi chez les femmes (on peut aussi penser que les hommes rechignent davantage à l'avouer mais c'est un autre débat).

Bref, notre système d'attribut est pour moi assez difficile à interpréter et appliquer de manière homogène. Une piste de réflexion pourrait être : 
01 points lumineux régulier tout les 20 m sur ce côté de la chaussée
02 points lumineux régulier entre 20 et 50 m sur ce côté de la chaussée
03 points lumineux irrégulier de ce côté de la chaussée
04 points lumineux régulier tout les 20 m de l'autre côté de la chaussée
05 points lumineux régulier entre 20 et 50 m sur de l'autre côté de la chaussée
06 points lumineux irrégulier de l'autre côté de la chaussée
07 non éclairé

J'ai des doutes sur les distances, c'est purement arbitral. Peut-être que Marion peut nous renseigner sur l'existence d'une norme, on pourrait s'y caler et éviter ainsi de multiplier les référentiels.

Autre doute c'est que nous ne prenons pas en compte la largeur de la rue si l'éclairage est situé de l'autre côté. Mais bon je ne voudrai pas multiplier les attributs possibles.

Un des avantages est que nous nous calons sur la philosophie du CNIG en ayant des attributs mesurables et tangibles et non pas des interprétations.

C'est sans doute perfectible

**Comment by Marion-Torterotot** [comment]
Je suis d'accord sur le caractère très arbitraire et subjectif de cet attribut, d'autant plus qu'on collecte de jour. On serait bien en peine juste de dire si l'éclairage fonctionne réellement (la présence de luminaires ne garantit pas cela) ! Si on ajoute à ça les pratiques de plus en plus courantes d'extinction des feux la nuit (entre 1h et 5h du matin souvent, mais pas toujours), ça devient très fluctuant !

Il y a effectivement des normes sur l'éclairage, notamment la NF EN 13201 sur l'éclairage dans les espaces publics, qui est très calée sur l'automobile (sans surprise) et qui est d'application volontaire, donc toutes les collectivités ne s'en préoccupent pas. La réglementation parle plus de lumen (quantité de lumière émise par le luminaire) que de lux (quantité de lumière reçue au sol). Sauf pour l'accessibilité, mais c'est limité aux abords des ERP/IOP (les fameux "20 lux en moyenne").
Les collectivités ont également des schémas directeurs d'éclairement (maintenant "schéma directeur d'aménagement lumière" SDAL), dans lesquels elles précisent les niveaux d'éclairement selon les types de voiries, voire rue par rue, en lux. Mais je n'arrive pas à savoir si ce document est obligatoire ou pas. Par ailleurs, ce sont des objectifs : pas toujours atteints, ou alors de façon hétérogène (notamment sur les places). Il faudrait mesurer pour vérifier, je ne sais pas si elles le font.
Pour plus d'infos, voici des fiches du Cerema sur le sujet : 
- explication de l'arrêté sur les nuisances lumineuses : https://doc.cerema.fr/digitalCollection/DigitalCollectionAttachmentDownloadHandler.ashx?parentDocumentId=15792&documentId=18256&skipWatermark=true&skipCopyright=true 
- explications des normes et réglementation en général : https://doc.cerema.fr/Default/doc/SYRACUSE/15792/aube-amenagement-urbanisme-biodiversite-eclairage-fiche-n-04-comprendre-l-arrete-ministeriel-du-27-d?_lg=fr-FR 
Il est possible de mesurer réellement le niveau d'éclairement, mais de ce que j'ai compris, ce sont des campagnes assez lourdes, avec un camion instrumenté (donc pas facile sur les espaces piétons, surtout s'ils sont grands et que le camion ne peut pas rouler dessus). Donc ça ne me semble pas souhaitable d'aller dans ce niveau d'exigence pour la collecte CNIG.

En bref, je suis pour la 2e proposition, **avec peut-être un doute sur ce que tu entends par "avec risque d'insuffisance" ?**

**Comment by antoine-tesson** [comment]
Risque d'insuffisance serait purement arbitraire et relève de l'impression visuelle du diagnostiqueur : vétusté, écart, orientation... Pas très scientifique mais plutôt un appel pour que la ville fasse une étude précise sur le tronçon

**Comment by Marion-Torterotot** [comment]
Ok, très arbitraire en effet... J'ai peur qu'on retombe sur le problème du début (adapté ou non adapté pour la déficience visuelle). C'est incomplet, mais ne vaudrait-il pas mieux se concentrer sur la présence des luminaires, sans présupposer de leur fonctionnement ?

01 présence de dispositifs d'éclairage réguliers
02 présence de dispositifs d'éclairage irréguliers
03 absence de dispositif d'éclairage visible

**Comment by Lenoxaic** [comment]
L'éclairage ne concerne pas que les déficients visuels et pour le coup nombreuses sont les personnes à rencontrer des problèmes vestibulaires. L'état du fonctionnement, risque d'insuffisance de l'éclairage entraine des problèmes d'équilibre dans les secteurs insuffisamment éclairés.

**Comment by GT-CNIG-DDU** [comment]
En faisant la synthèse des échanges, peut-on considérer que la proposition conjointe de @antoine-tesson et de @Marion-Torterotot convient à tous ? : 

> présence de dispositifs d'éclairage réguliers
> présence de dispositifs d'éclairage irréguliers
> absence de dispositif d'éclairage visible

En remplacement de : 

![image](https://github.com/user-attachments/assets/8523c993-d2f5-4b36-b1b1-8f85bc415326)

Post-scriptum : Ces valeurs initiales sont celles du [profil NeTEx accessibilité France](https://normes.transport.data.gouv.fr/normes/netex/accessibilite/). La question est donc à étudier en coordination avec les experts NeTEx (poke @TuThoThai).

**Comment by TuThoThai** [comment]
Pour note, NeTEx différencie les équipements (la méthode) d'éclairage et l'appréciation globale de l'éclairage

![Image](https://github.com/user-attachments/assets/641b23a2-534e-46de-b9d4-923e29b8ab2d)

**Comment by antoine-tesson** [comment]
Comme vu au dernier GT, ce sujet n'est pas clos et effectivement la proposition de David est la plus à même de récolter des info qui ne laissent pas de place à l'intrétation.

Néanmoins, pour coller au mieux avec Netex (car la compatibilité avec Netex est quand même un sujet majeur), il me semble opportun de bien partir là dessus : 

présence de dispositifs d'éclairage réguliers = welllit
présence de dispositifs d'éclairage irréguliers = poorlyliy
absence de dispositif d'éclairage visible = unlit

Je ne pense pas qu'on risque non plus d'avoir des divergences énormes entre les collecteurs et je pense aussi qu'on peut donner des règles dans le guitbook.

**Comment by Lenoxaic** [comment]
Je suis d'accord avec la proposition de David et antoine ça permet de récolter des infos et d'être compatible Netex.
Il y a une grande part de perception dans cette problématique d'éclairage. Cela demande d'autres investigations.
Sylvaine

**Comment by GT-CNIG-DDU** [comment]
Merci @Lenoxaic, mais comme ce ne sont pas tout à fait les mêmes propositions l'idéal de parvenir à une proposition qui fasse consensus.  ; - ) 
Ou bien tu suggères de continuer à investiguer ?

**Comment by Marion-Torterotot** [comment]
Bonjour. Même avis que Sylvaine. La proposition n'est pas 100% satisfaisante, mais elle a le mérite d'être simple et compatible avec NeTEx.
Bonne journée !

**Comment by GT-CNIG-DDU** [comment]
Je suis du même avis que @TuThoThai, @Marion-Torterotot, @antoine-tesson, @Lenoxaic : simple et compatible NeTEx, et toujours améliorable ultérieurement si nécessaire.

J'attends de connaître ton avis @AMIAUD1 : es-tu également d'accord avec cette proposition ?

**Comment by AMIAUD1** [comment]
Bonjour à tous, 
je vous propose une légère variante : 
- Circulation avec dispositif d’éclairage = welllit (si candélabre sur le trottoir)
- Circulation avec éclairage indirect = poorlyliy (si candélabre est sur un autre trottoir dans la même rue ou irrégulier)
- Circulation non éclairée = unlit (aucun dispositif d'éclairage)

L'observation de la régularité ou l'irrégularité est, me semble-t-il, sujet à interprétation par un auditeur. Il sera préférable de rester sur ce qui est visible ou non. Un champ sur la qualité de l'éclairage et la régularité de la luminosité devrait ensuite être analysée à l'aide des données des plans lumières des communes.

Bonne journée à tous

**Comment by Marion-Torterotot** [comment]
Bonjour,
Je tente une 3e option (désolée...) : 
-     dispositif d'éclairage direct et rapproché
-     dispositif d'éclairage indirect ou espacé
-     absence de dispositif d'éclairage

Je ne suis pas sûre que la notion d'éclairage direct ou indirect soit plus simple que celle de la régularité. Mais on peut avoir un dispositif régulier qui soit pour autant insuffisant car il n'y a pas assez de candélabres (au total) ou parce qu'ils sont orientés pour éclairer la route plutôt que le trottoir. Donc je propose une notion "spatiale" à la place.
Je pense que ça ne résout pas totalement le problème, mais ça peut être un compromis pour cette nouvelle version du standard. Il faudra, de toute façon, l'expliciter dans le gitbook.

Qu'en pensez-vous ?

**Comment by AMIAUD1** [comment]
Je suis d'accord avec la proposition de Marion.
Bonne fin de journée

**Comment by GT-CNIG-DDU** [comment]
Merci @AMIAUD1.
Si tout le monde est d'accord, je propose que l'on retienne cette proposition pour le standard CNIG Accessibilité rev. 2025-03.

**Comment by antoine-tesson** [comment]
Je n'arrive pas à être totalement d'accord. Eclairage indirect ne veut pas forcement dire que l'éclairage est mauvais. Je ne vois du coup pas bien ce que ça apporte. De plus, il semblait que l'avantage était d'être conforme aussi à Netex... l'est-on toujours avec ta proposition?

**Comment by TuThoThai** [comment]
Pour répondre à @antoine-tesson : on est conforme NeTEx tant que les valeurs d'export après collecte suivant le standard CNIG reste les valeurs : 
- wellLit
- poorlyLit
- unlit
- unknown
- other

Le but est donc de se mettre d'accord sur ce qu'on met dans le standard CNIG face à ces valeurs d'export et en français fans le texte. 😄 

Pour ce qui est des trois propositions, et si on parle bien de trottoirs, j'aurais eu tendance à suggérer : 
- wellLit : éclairage régulièrement espacé, constamment alimenté et dirigé vers le trottoir
- poorlyLit : éclairage non régulièrement existant ou à alimentation intermittente et/ou non dirigé vers le trottoir
- unlit : pas d'éclairage du trottoir
- unknow : inconnu
- other : autre

Ma propisition est pour essayer de prendre en compte la situation dans laquelle c'est la route qui est éclairée mais qu'on n'y voit rien sur le trottoir. Ou alors certaines portions de rues qui ne s'éclairent que si la présence de véhicules est détectée sur la route... 


_Note générale : j'ai corrigé certains de vos commentaires pour rajouter les 2 majuscules à NeTEx_

**Comment by TuThoThai** [comment]
Pour répondre à @antoine-tesson : on est conforme NeTEx tant que les valeurs d'export après collecte suivant le standard CNIG reste les valeurs : 
- wellLit
- poorlyLit
- unlit
- unknown
- other

Le but est donc de se mettre d'accord sur ce qu'on met dans le standard CNIG face à ces valeurs d'export et en français fans le texte. 😄 

Pour ce qui est des trois propositions, et si on parle bien de trottoirs, j'aurais eu tendance à suggérer : 
- wellLit : éclairage régulièrement espacé, constamment alimenté et dirigé vers le trottoir
- poorlyLit : éclairage non régulièrement existant ou à alimentation intermittente et/ou non dirigé vers le trottoir
- unlit : pas d'éclairage du trottoir
- unknow : inconnu
- other : autre

Ma propisition est pour essayer de prendre en compte la situation dans laquelle c'est la route qui est éclairée mais qu'on n'y voit rien sur le trottoir. Ou alors certaines portions de rues qui ne s'éclairent que si la présence de véhicules est détectée sur la route... 


_Note générale : j'ai corrigé certains de vos commentaires pour rajouter les 2 majuscules à NeTEx_

**Comment by AMIAUD1** [comment]
Bonjour, 
la proposition de @TuThoThai est très bien. Nos réponses tournent autour de la même approche et la synthèse @TuThoThai  est totalement intégrable pour faire de la collecte sans possibilité d'effectuer des mesures en lux.
Bonne journée à vous

**Comment by GT-CNIG-DDU** [comment]
Merci pour cette synthèse @TuThoThai. Les valeurs étant un peu longues, il conviendra de distinguer les valeurs de leur définition : 

![Image](https://github.com/user-attachments/assets/8f4daa1e-f5d7-492d-8142-af9cac20bb46)
